docs/src/user_guide/sample_data.md

```diff
@@ -0,0 +1,82 @@
+# [Tutorial models](@id sample_data)
```

Comment:
I think this section needs a bit more work/description. It is now just a list of the different test models without further explanation. Also there is overlap with the section (https://deltares.github.io/Wflow.jl/previews/PR184/user_guide/model-setup/#Building-a-model-from-scratch). Do we want to include a real tutorial on models here, or just provide some sample data from our test bank (with some more description)?

Reply:
Given that the other steps in the user guide are already a small tutorial, perhaps we can rename this section to (something along the lines of) `Example models`? A user can explore these input files combined with the TOML to understand the requirements for the different model types.

Reply:
Yes, sounds good, including a bit more description (see also model-setup.md) and remove this section from model-setup.md.

Reply:
Moved the text from model-setup.md to sample_data.md, and added a bit more information.